relay-dynamic-config/src/global.rs

```diff
@@ -13,6 +15,28 @@ pub struct GlobalConfig {
     pub measurements: Option<MeasurementsConfig>,
 }
 
+impl GlobalConfig {
+    /// Returns the full filepath of the global config.
+    fn path(base: &Path) -> PathBuf {
+        base.join("global_config.json")
```

Comment:
I think what @jan-auer  meant is just remove the function and use the join in the `load` function.  Since it's used in only one place, from what I can see.